vwifi.c

```diff
@@ -18,18 +18,19 @@ MODULE_DESCRIPTION("virtual cfg80211 driver");
 
 #define MAX_PROBED_SSIDS 69
 
-#ifndef DEFAULT_SSID_LIST
-#define DEFAULT_SSID_LIST "[MyHomeWiFi]"
-#endif
-
 #define SCAN_TIMEOUT_MS 100 /*< millisecond */
 
+#define MAC_FMT "%02x:%02x:%02x:%02x:%02x:%02x"
+#define MAC_PRINT(a) a[0], a[1], a[2], a[3], a[4], a[5]
```

Comment:
I think you can replace MAC_FMT with "%pM".

Reply:
OK. I have removed `MAC_FMT` and just use `%pM`, it makes our code more cleaner!